sections/plate_notation.tex

```diff
@@ -14,51 +14,27 @@ \subsection{Author-Topic \gls{lda} and Category-Topic \gls{lda}}
 
-\begin{figure*}[ht]
-	\centering
-	\begin{subfigure}{0.275\textwidth}
-		\centering
-		\resizebox{\textwidth}{!}{%
-			\input{figures/author_plate_original}
-		}
-		\caption{Author \gls{lda} from \cite{author_topic_2012}.}
-		\label{fig:original_author_lda}
-	\end{subfigure}
-	\hspace{2em}
-	\begin{subfigure}{0.275\textwidth}
-		\centering
-		\resizebox{\textwidth}{!}{%
-		\input{figures/category_plate}
-		}
-		\caption{Category \gls{lda}.}
-		\label{fig:category_lda}
-	\end{subfigure}
-	\hspace{2em}
-	\begin{subfigure}{0.275\textwidth}
-		\centering
-		\resizebox{\textwidth}{!}{%
-			\input{figures/author_plate}
-		}
-		\caption{Author \gls{lda}.}
-		\label{fig:author_lda}
-	\end{subfigure}	
-	\caption{Plate notation for the metadata \gls{lda} models.}
-	\label{fig:metadata_lda}
-\end{figure*}
-
 \subsection{Pachinko Allocation}
 In order to handle the hierarchical structure of the taxonomy metadata field, we use a hierarchical topic model, namely the \acrfull{pam} from \citet{li2006pachinko}.
-
-In this paper, we construct a five-level \gls{pam}, following the format presented by \citet{li2006pachinko}.
-The \gls{dag} structure used in this paper is in \autoref{fig:pachinko_dag}, where each layer has been named.
-The plate notation for this model is shown in \autoref{fig:pachinko}
-
-Some layers in our \gls{dag} structure are duplicates of layers from the taxonomy field within our dataset.
-This allows us to have some nodes represent a topic based on a specific taxonomy.
+Pachinko allocation generalizes \gls{lda}, making it possible to construct topic hierarchies based on any \gls{dag} structure.
+\gls{pam} is a topic model focusing on finding topics of different abstraction levels and modeling the connections between these topics.
+
+Each node in the \gls{dag} structure represents a topic in the pachinko allocation model. 
+However, unlike \gls{lda} where topics are distributions over words, in \gls{pam} topics are multinomial distributions over words and/or other topics further down the hierarchy of the \gls{dag} structure.
+An example of the \gls{dag} structure used in this paper is in \autoref{fig:pachinko_dag} where each level of the \gls{pam} is shown.
+The idea behind the \gls{dag} structure is to be able to model correlations between topics and in turn make more coherent topics.
+  
+In this paper, we use a layered \gls{pam}, as in \cite{author_topic_2012}, meaning that the \gls{dag} structure is divided into layers where each layer is fully connected.
+However, \citet{li2006pachinko} use four layers where we use five to capture more taxonomy fields.
+
+We replicate some layers in our \gls{dag} structure based on the structure from the taxonomy field within our dataset, having some nodes represent a topic based on a specific taxonomy.
+An example of this could be in the second layer, we have "STEDER", and this is connected to "Danmark" in the third layer.
```

Comment:
```suggestion
An example of this can be seen in \autoref{fig:pachinko_dag}, where we have the node "STEDER", and this is connected to "Danmark" in the third layer.
```